dnf-behave-tests/features/steps/transaction.py

```diff
@@ -207,6 +207,7 @@ def parse_microdnf_transaction_table(lines):
     lines = lines[:transaction_end]
 
     label_re = re.compile(r"^([^ ].+):$")
+    replacing_re = re.compile(r"^(?P<label>replacing) +(?P<nevra>[^ ]*)$")
```

Comment:
The `label` regex group is not used, so it can be removed from the regex... it's a good idea to keep the regexes as simple as possible, they're hard to read enough already :slightly_smiling_face:

Reply:
A remnant from another regex. Fixed.